Services/communication_service.h

```diff
@@ -21,14 +21,123 @@
 #include <stdio.h>
 
 #include "services.h"
-#include "service_response.h"
-#include "service_utilities.h"
-#include "system.h"
+
+
+#define MAX_W_CMDLEN 120 //from uTransceiver.h
+#define STAT_WORD_LEN 12
 
 struct temp_utc {
   uint32_t sensor_temperature;
 };
 
+typedef struct __attribute__((packed)) {
+    uint8_t status;
+    uint8_t mode;
+} Sband_PowerAmplifier;
+
+typedef struct __attribute__((packed)) {
+    uint8_t scrambler;
+    uint8_t filter;
+    uint8_t modulation;
+    uint8_t rate;
+} Sband_Encoder;
+
+typedef struct __attribute__((packed)) {
+  float freq;
+  uint8_t PA_Power;
+  Sband_PowerAmplifier PA;
+  Sband_Encoder enc;
+} Sband_config;
+
+typedef struct __attribute__((packed)) {
+    uint8_t PWRGD;
+    uint8_t TXL;
+} Sband_Status;
```

Comment:
I'm thinking these should probably go into the HAL header files - thoughts?

Reply:
Just because the HAL depends on these things more directly than the service code (I would argue)

Reply:
Yeah, initially, we weren't planning on using all of these structs in HAL, and I was following the format from other services. They're both using them I'd say. I could see them in any of the header files. 
One advantage of moving them to HAL headers would be to split S-band and UHF structs. Now, each one is including the others for their HAL without using them but the service file uses them both.
There won't be much left in the service header file though if we remove this.

Reply:
That's fine, nothing wrong with a 2-line header